flink-python/setup.py

```diff
@@ -317,13 +317,13 @@ def extracted_output_files(base_dir, file_path, output_directory):
         'pyflink.bin': ['*']}
 
     install_requires = ['py4j==0.10.9.7', 'python-dateutil>=2.8.0,<3',
-                        'apache-beam>=2.43.0,<2.49.0',
+                        'apache-beam>=2.52.0',
```

Comment:
Should there be an upper bound for dependency version to prevent breaking changes coming in?
Or can we make this version fixed (use `==`)?